src/components/phone/AgentBoard/Board.vue

```diff
@@ -44,14 +44,14 @@ export default {
   computed: {
     lang() {
       return this.getLang({
-        currentCall: '~~Current Call',
-        scripts: '~~Scripts',
-        calltime: '~~Call Time',
-        english: '~~English',
-        callStat: (calls, days) => `~~${calls} Calls | ${days} Days`,
-        notes: '~~Notes',
-        callStatus: '~~Call Status *',
-        issuesResolved: '~~Issues resolved',
+        currentCall: this.$t('boardVue.current_call'),
+        scripts: this.$t('boardVue.scripts'),
+        calltime: this.$t('boardVue.call_time'),
+        english: this.$t('boardVue.english'),
+        callStat: (calls, days) => `~~${calls} this.$t('boardVue.calls') | ${days} this.$t('boardVue.days')`,
+        notes: this.$t('boardVue.notes'),
+        callStatus: this.$t('boardVue.call_status'),
+        issuesResolved: this.$t('boardVue.issues_resolved'),
```

Comment:
Same thing for this.

Working on migrating the rest of the app to use getLang, but for now just dont add `this.$t` if it wasnt there already.